schedule/yast/lvm/lvm+cancel_existing_cryptlvm_pvm.yaml

```diff
@@ -0,0 +1,34 @@
+---
+description: >
+  Conduct installation cancelling encrypted partitions activation, and creating
+  non-encrypted lvm setup from scratch. Using pre-partitioned disk image to validate
+  encrypted partitions activation and that we can re-ecnrypt the disk.
```

Comment:
s/re-ecnrypt/re-encrypt/, also existing comment in `lvm+cancel_existing_cryptlvm.yaml`

Reply:
is it [here](https://openqa.suse.de/tests/4211491#step/lvm_ignore_existing/4) where we should re-encrypt the disk to follow this description? In the original job I think this was not done, probably makes sense to do the step, wdyt?

Reply:
> is it [here](https://openqa.suse.de/tests/4211491#step/lvm_ignore_existing/4) where we should re-encrypt the disk to follow this description? In the original job I think this was not done, probably makes sense to do the step, wdyt?

Seems like the description is wrong. Another description is used in Test Suite settings: https://openqa.suse.de/tests/4200298 (`Test suite cancels encrypted partitions activation and performs installation with unencrypted lvm.`)

The case when test suite cancels encryption and performs encrypted installation already covered in [cryptlvm+cancel_existing](https://openqa.suse.de/tests/4200894).

So, I used the description from Test Suite settings. Also, I changed it for 'lvm+cancel_existing_cryptlvm' on other architectures.